package.json

```diff
@@ -84,6 +85,8 @@
     "classnames": "^2.2.5",
     "debug": "^3.1.0",
     "lodash.throttle": "^4.1.1",
+    "mkdirp": "^0.5.1",
```

Comment:
@molnosikristof can you remove these from here and put into `devDependencies`?

Reply:
Done

Reply:
and also the `node-resemble-js` pls